From 0bc517ed6b7de41dfe672bc43a4a23d338ce0285 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Sun, 15 Jul 2018 06:38:00 -0500
Subject: [PATCH] "0001-rr_variant-unofficial-meticulus
 _______________________________

*Previous From d4b82c458f01b19a4432fe61f8d1ce2c8c8319ad Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Sun, 15 Jul 2018 06:32:57 -0500
*Previous Subject: [PATCH] "0001-rr_variant-unofficial-meticulus
*_______________________________
*
**Previous From ab5b8e8d266f033b29914ef3a112ed543f7771f9 Mon Sep 17 00:00:00 2001
**Previous From: Meticulus <theonejohnnyd@gmail.com>
**Previous Date: Fri, 2 Mar 2018 08:14:43 -0600
**Previous Subject: [PATCH] "0001-rr_variant-unofficial-meticulus
**_______________________________
**
***Previous From ec540fdba22a60700c5cf157fb1bf74f7f37e507 Mon Sep 17 00:00:00 2001
***Previous From: Meticulus <theonejohnnyd@gmail.com>
***Previous Date: Wed, 28 Feb 2018 09:18:21 -0600
***Previous Subject: [PATCH] "0001-rr_variant-unofficial-meticulus
***_______________________________
***
****Previous From 0ea095c8b583920efa9f13f499cc26537d50d2c2 Mon Sep 17 00:00:00 2001
****Previous From: Meticulus <theonejohnnyd@gmail.com>
****Previous Date: Mon, 27 Nov 2017 08:17:34 -0600
****Previous Subject: [PATCH] "0001-rr_variant-unofficial-meticulus
****_______________________________
****
*****Previous From 62827aa279c220ff8eb7dc72f653392906985787 Mon Sep 17 00:00:00 2001
*****Previous From: Meticulus <theonejohnnyd@gmail.com>
*****Previous Date: Sun, 18 Jun 2017 06:54:24 -0500
*****Previous Subject: [PATCH] rr_variant: unofficial -> meticulus
*****""
***
***Change-Id: I3fc5c0e76abbdb03c7df2711e6de98388d653aa0"
**
**Change-Id: Iaae98563df0c3e7f00a6ee34bd37f195d177480d"
*
*Change-Id: I7e2c698db5bf9aaddcb1f6844b5810f1ddeda281"

Change-Id: I5255ba67b7f325fff40f126324829a12a495d371
---
 config/common.mk    | 3 +--
 tools/changelog.sh  | 2 +-
 tools/rr_root.sh    | 6 +++++-
 tools/rr_variant.sh | 8 ++++----
 4 files changed, 11 insertions(+), 8 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 6e3d878..302ddc1 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -172,8 +172,7 @@ PRODUCT_PACKAGES += \
     WeatherProvider \
     OmniJaws \
     OmniStyle \
-    ResurrectionOTA \
-    ResurrectionStats
+    ResurrectionOTA
 
 # Berry styles
 PRODUCT_PACKAGES += \
diff --git a/tools/changelog.sh b/tools/changelog.sh
index 8ab0b0b..1e09968 100755
--- a/tools/changelog.sh
+++ b/tools/changelog.sh
@@ -51,7 +51,7 @@ echo -e "";
 echo -e ${cya}" ▼ Type a number"${txtrst}
 echo -e "";
 # use 'export days_to_log=5' before '. build/envsetup.sh' were 5 is days to log
-read -r -t 30 days_to_log || days_to_log=7
+days_to_log=0
 fi
 echo >> $Changelog;
 echo " ▼ $source_name Ver 6.1.0 Changelog"    >> $Changelog;
diff --git a/tools/rr_root.sh b/tools/rr_root.sh
index 4f0503b..cef3cec 100755
--- a/tools/rr_root.sh
+++ b/tools/rr_root.sh
@@ -34,7 +34,11 @@ echo -e "";
 echo -e ${cya}" ▼ Pick a number"${txtrst}
 echo -e "";
 $normal
-read -t 30 askvariant
+if [[ "$WITH_SU" == "true" ]]; then
+askvariant=0
+else
+askvariant=1
+fi
 sleep 1
 if [ "$askvariant" == "0" ];
 then
diff --git a/tools/rr_variant.sh b/tools/rr_variant.sh
index 113cf00..923e578 100755
--- a/tools/rr_variant.sh
+++ b/tools/rr_variant.sh
@@ -24,7 +24,7 @@
 
 echo -e ${ylw}"\n\n ▼ Which build-type do you want to choose?\n"${txtrst}
 echo -e "";
-echo -e ${blu}" 〉 1- Unofficial "${txtrst}${red}"  ︱ For the unofficial builds"${txtrst}
+echo -e ${blu}" 〉 1- CodeOfHonor "${txtrst}${red}" ︱ For the CoH builds"${txtrst}
 echo -e "";
 echo -e ${blu}" 〉 2- Nightly "${txtrst}${red}"     ︱ Only for the official daily builds by the maintainers"${txtrst}
 echo -e "";
@@ -43,13 +43,13 @@ echo -e "";
 echo -e ${cya}" ▼ Pick a number"${txtrst}
 echo -e "";
 $normal
-read -t 30 askvariant
+askvariant=1
 sleep 1
 if [ "$askvariant" == "1" ]
 then
 echo -e "";
-echo -e ${blu}" ▪ Building unofficial variant "${txtrst}
-export RR_BUILDTYPE=Unofficial
+echo -e ${blu}" ▪ Building CodeOfHonor variant "${txtrst}
+export RR_BUILDTYPE=CodeOfHonor
 echo -e "";
 echo -e "";
 $normal
-- 
2.7.4

